tests/integration/targets/cloudwatchevent_rule/tasks/main.yml

```
@@ -0,0 +1,105 @@
+---
+
+- module_detauls:
+  group/aws:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    security_token: "{{ security_token | default(omit) }}"
+    region: "{{ aws_region }}"
+
+  block:
+    - name: create cloudwarch event rule for integration test
+      register: result
+      cloudwatchevent_rule:
+        name: run_foo
```

We run a lot of these tests in parallel (multiple versions of Ansible and Python) as such we need to use a unique name for each test to avoid things clashing.

```suggestion
        name: '{{ resource_prefix }}'
```